sass/base/components/card-block/_card.scss

```diff
@@ -19,6 +19,11 @@
   @include breakpoint($screen-md) {
     margin: 0 gutter() 30px;
   }
+  &[class*="bg--"] {
```

Comment:
should we use `class^=` (anything **starting** with `bg--`). `class*=` will select anything that contains `bg--`.

Reply:
yeah that's what I thought too, but if I use `^=` it requires that the class attribute value starts with that. So that won't work as the `bg--colour` class will always be preceded by the `card` class.

Reply:
I think for the time being we should put this in, so the padding works again. But I will put in a ticket to solve this properly and give the card a no-bg-colour class.

Reply:
ok